pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   # plugin_platform_interface depends on meta anyway
   meta: ^1.3.0
   stack_trace: ^1.12.0
+  dio: ^5.9.0
```

Comment:
this has to be a new package within the posthog-flutter repo so users that dont use dio dont need to have this as a dependency

Reply:
Updated.